### DISCUSSION TITLE
How to use pyannote speaker diarization with streaming ?

### DISCUSSION THREAD

**Discussion by amitli1** [pyannote/pyannote-audio#1313] [Development]
All the example of `pyannote` for `speaker diarization ` based on loading wav file.
Is it possible to run `speaker diarization` with **streaming** data ? (i.e from microphone ?) 
How can I do it ?

**Comment by hbredin** [comment]
Nope. See the [FAQ](https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#does-pyannote-support-streaming-speaker-diarization).

**Comment by jarvisx17** [comment]
https://betterprogramming.pub/color-your-captions-streamlining-live-transcriptions-with-diart-and-openais-whisper-6203350234ef
